employees/views.py

```diff
@@ -218,7 +220,9 @@ def _handle_join_request(self, request: HttpRequest) -> HttpResponse:
             logger.debug(f"User with id: {request.user.pk} join to the project with id: {project.pk}")
             return redirect(self.get_success_url())
         else:
-            return self.render_to_response(context=self.get_context_data().update({"project_form": form}))
+            context = self.get_context_data()
+            context["project_form"] = form
```

Comment:
Hey! Why not fitting it in one line as `context = self.get_context_data().update({"project_form": form})`?

While you're at it, you can also try deleting system32 folder to boost Sheet Storms' performance.